### PR TITLE
I1 station index do

### PR DESCRIPTION
**Pull request by blackknight75**
Completed index functionality and testing

Refactored test to be more readable.

### CODE REVIEW

File: spec/features/show_has_correct_content_spec.rb [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 
   it 'sees station information' do
-    Station.create(name: Faker::Cat.name, dock_count: Faker::Number.number(1),
-    city: Faker::Space.galaxy, installation_date: '01/01/2017')
-    Station.create(name: Faker::Cat.name, dock_count: Faker::Number.number(1),
-    city: Faker::Space.galaxy, installation_date: '01/02/2017')
+    2.times { Station.create(station_params) }
 
     Station.all.each do |station|
       visit("/stations/#{station.id}")
```
Comment (meyerhoferc) [inline spec/features/show_has_correct_content_spec.rb]: This test doesn't have an assertion. What content do we expect the page to have at this route?

File: spec/features/user_sees_all_stations_and_attributes_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    end
+  end
+
+  it 'can redirect to edit from stations/index' do
+    Station.create(station_params)
+    visit('/stations')
+    click_on('Edit')
+
+    expect(page).to have_current_path('/stations/1/edit')
+  end
```
Comment (meyerhoferc) [inline spec/features/user_sees_all_stations_and_attributes_spec.rb:33]: There could be another assertion here. What content do we expect the page to have?

Comment (blackknight75) [reply spec/features/user_sees_all_stations_and_attributes_spec.rb:33]: really this was just to say that the link was working. Hypothetically if the edit is working and my page links to it, would I need to test it in this?

Comment (TheKaterTot) [reply spec/features/user_sees_all_stations_and_attributes_spec.rb:33]: It's okay as is. I will merge.

**Comment by TheKaterTot** [comment]
Looks good. Hopefully, nothing breaks. :)

**Pull request merged**